src/Organic/Organic.php

```diff
@@ -1312,6 +1337,16 @@ public function syncPluginConfig() {
                 $this->updateOption( 'organic::sentry_dsn', $sentryDSN, false );
                 $this->configureSentryForSite();
             }
+            if ( $config['triggerContentResync'] && ! $this->contentResyncTriggeredRecently() ) {
+                global $wpdb;
+                $wpdb->get_results(
+                    $wpdb->prepare(
+                        "UPDATE $wpdb->postmeta SET meta_value = 'unsynced' WHERE meta_key = %s",
+                        SYNC_META_KEY
+                    )
+                );
+                $this->updateContentResyncStartedAt();
```

Comment:
I wonder if these lines should be pulled out into separate function. Not really dead set on this.